web/src/main.js

```diff
@@ -17,7 +17,7 @@ const HMRApp = hot(module)(App)
 
 ReactDOM.render(<HMRApp />, container)
 
-if (buildConfig().splashScreen) {
+if (buildConfig().web.splashScreen) {
```

Comment:
Is it possible to do `buildConfig().native` here? If yes then maybe only `web` should be available.

Reply:
No, since there is no `.native` in the build config. It is possible to access `.ios` and `.android` though. You introduced this with firebase/google config and I think it is okay like this. What do you think?